gluecodium/src/main/java/com/here/gluecodium/validator/LimeValidatorUtils.kt

```diff
@@ -25,6 +25,8 @@ import com.here.gluecodium.model.lime.LimeNamedElement
 import com.here.gluecodium.model.lime.LimeType
 
 internal object LimeValidatorUtils {
+    const val LIME_MARKDOWN_DOCS = "https://github.com/heremaps/gluecodium/blob/master/docs/lime_markdown.md"
```

Comment:
I put it here to create a follow-up PR, which will embed the full link to the docs also in the case of `LimeLambdaValidator` and `LimeFunctionsValidator`.